pkg/cnab/config-adapter/stamp.go

```diff
@@ -102,9 +102,6 @@ func (c *ManifestConverter) DigestManifest() (string, error) {
 	v := pkg.Version
 	data = append(data, v...)
 
-	// (for instance, during publish), so add this to the data
-	data = append(data, c.Manifest.Image...)
```

Comment:
Sorry I know we talked about this but I've gotten fuzzy over the holidays. Can you remind me why we can remove this? I am just wondering if the info is now in the generate manifest, or if we can't put it in anymore because the flag may have changed the value?

Reply:
I/we had added this previously to automatically trigger a rebuild whenever a bundle's image was updated.  (Say, when a new registry or tag value is supplied on publish.)  However, in this PR, instead of rebuilding the bundle, we re-tag images/bundle references and use these for publishing.

As noted elsewhere, it does mean that the actual contents (bundle.json/porter.yaml) included in the installer/invocation image may have different registry/reference values from the final published versions.  For instance, the built bundle might have `registry: myregistry.com` but if I publish the bundle via `porter publish --registry localhost:5000`, the published invocation image (`localhost:5000/mybuns:v0.1.0`) will have a `porter.yaml` file with the original `registry: myregistry.com` field.

Reply:
Thanks for clearing that up. I know some users want to be able to publish to alternate locations, either because they moved it across an airgap or something else, and have the bundle _not_ contain the original information. One workaround in that case would be to use non-sensitive information when building the bundle and then always set a --registry, etc during publish.